models/auth/auth_token.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// Copyright 2023 The Forgejo Authors. All rights reserved.
```

Comment:
```suggestion
// Copyright 2023 The Gitea Authors. All rights reserved.
```

Reply:
Is it a requirement to remove copyright notices to contribute code to Gitea?

Reply:
That would require permission from the author(s) of the code.

Reply:
This has been explained many times already to you, and you have been warned as well in the last pull request to follow contribution guidelines. If you'd like to adhere to the contribution guidelines we'd be happy to have any contribution, however as you've shown to willfully disregard them action may need to be taken.

Reply:
where, in the contributions guidelines, can I find the requirement to remove an existing copyright notice from a MIT licensed code?

Reply:
We're not asking you to remove copyright, but instead asking for copyright assignment. If each of our contributors added to the headers of the file they touched then some files could have over 1000 listings, this is clearly untenable. We don't remove copyright, for example there is places where we used code from go and kept their headers because we took that code and adhered to the license, but as you are contributing the code that's where the difference is. If you don't have the rights to do so under DCO and contribution document then that prevents us from accepting code. This is following guidance from the Linux Foundation to ensure a codebase free of any conflict.

Reply:
I was not aware Gitea required a copyright assignment. Could you please let me know where this is explained in the contributions guidelines?

Reply:
For the past 7 years, Gitea has required the same policy on copyright headers; you've been told this in other PRs, and have modified your PRs to be accepted. Not adhering to it once or twice can be forgiven as an accident, however, continuing to do so, including after being warned, has to be treated as you not acting in good faith, especially in this PR for a security-related PR knowing it can't be merged as-is.
Thankfully we've been working on a PR for this as well, and we were going to send it over privately once ready, so we are able to resolve this issue via a different PR without code that is tainted by copyright issues.
As I've said before, we welcome all contributions as long as they adhere to the project standards, if you/Forgejo are refusing to do so that's unfortunate.

Reply:
> I was not aware Gitea required a copyright assignment. Could you please let me know where this is explained in the contributions guidelines?

It has been documented 7 years ago: https://github.com/go-gitea/gitea/commit/a90b25226a4dae41c439f4d4919ef07e21b2ad43?short_path=eca12c0#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055

![image](https://github.com/go-gitea/gitea/assets/2114189/1445fce2-a522-43a0-a5a6-2daef4832e5a)


And it has been updated to use SPDX format: https://github.com/go-gitea/gitea/blame/16a766cba10dcaf53cc413cee62a952031bef5bc/CONTRIBUTING.md#L430

![image](https://github.com/go-gitea/gitea/assets/2114189/828da126-9d3e-48bf-9516-bb9264f9b946)